rtsp-to-web/config.yaml

```diff
@@ -21,6 +21,10 @@ options:
   http_port: 8083
   rtsp_port: 5541
   insecure_skip_verify: false
+  ice_servers:
+    - stun:stun.l.google.com:19302
+  ice_username: ""
+  ice_credential: ""
 schema:
```

Comment:
Does the schema also need to be updated so this can show up in the UI?

Reply:
Valid point. I will update

Reply:
I don't think this happened? I will add this.

Reply:
Oh I forgot, this was reverted...